.github/workflows/ci.yml

```diff
@@ -923,3 +923,43 @@ jobs:
     - name: run ./ci/docfx.cmd
       shell: cmd
       run: ./ci/docfx.cmd
+
+  w3c_trace_context_compliance_v1:
+    name: W3C Distributed Tracing Validation V1
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout open-telemetry/opentelemetry-cpp
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+      - name: setup
+        env:
+          CC: /usr/bin/gcc-10
+          CXX: /usr/bin/g++-10
+          PROTOBUF_VERSION: 21.12
```

Comment:
Is `PROTOBUF_VERSION` needed here?

Reply:
Ah, I copied the steps from an existing job and edited it, so this was probably left over. I doubt it's required. I can try to remove it and see if CI still works.

It should, I think.

Reply:
Removed this extra env variable, thanks.